app/views.py

```diff
@@ -0,0 +1,204 @@
+# app/__init__.py
+from flask_api import FlaskAPI
+from flask_sqlalchemy import SQLAlchemy
+from flask import request, jsonify, abort, make_response
+
+from instance.config import app_config
+
+
+# initialize sql-alchemy
+db = SQLAlchemy()
+
+
+def create_app(config_name):
+    import decorator
+    from app.models import BucketList, BucketListItem
+
+    app = FlaskAPI(__name__, instance_relative_config=True)
+    app.config.from_object(app_config[config_name])
+    app.config.from_pyfile("config.py")
+    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    db.init_app(app)
+
+    @app.route("/bucketlists/", methods=["POST", "GET"])
+    @decorator.auth_token
+    def bucketlists(*args, **kwargs):
```

Comment:
see if you can use flask views for bucketlists and items as used in auth. the methods in auth look cleaner :)